core/slim_globals.h

```diff
@@ -354,6 +354,432 @@ extern EidosValue_String_SP gStaticEidosValue_StringT;
+"            \"index\": 0,\n"
+"            \"type\": \"integer\"\n"
+"        }\n"
+"    },\n"
+"    \"required\": [\n"
+"        \"slim_id\",\n"
+"        \"is_null\",\n"
+"        \"genome_type\"\n"
+"    ],\n"
+"    \"type\": [\n"
+"        \"object\",\n"
+"        \"null\"\n"
+"    ]\n"
+"}";
+
+const std::string SLIM_TSK_INDIVIDUAL_METADATA_SCHEMA =
+"{\n"
+"    \"$schema\": \"http://json-schema.org/schema#\",\n"
+"    \"additionalProperties\": false,\n"
+"    \"codec\": \"struct\",\n"
+"    \"description\": \"SLiM schema for individual metadata.\",\n"
+"    \"examples\": [\n"
+"        {\n"
+"            \"age\": -1,\n"
+"            \"flags\": 0,\n"
+"            \"pedigree_id\": 123,\n"
+"            \"population\": 0,\n"
+"            \"sex\": 0\n"
+"        }\n"
+"    ],\n"
+"    \"flags\": {\n"
+"        \"SLIM_INDIVIDUAL_METADATA_MIGRATED\": {\n"
+"            \"description\": \"Whether this individual was a migrant, either in the generation when the tree sequence was written out (if the individual was alive then), or in the generation of the last time they were Remembered (if not).\",\n"
+"            \"value\": 1\n"
+"        }\n"
+"    },\n"
+"    \"properties\": {\n"
+"        \"age\": {\n"
+"            \"binaryFormat\": \"i\",\n"
+"            \"description\": \"The age of this individual, either when the tree sequence was written out (if the individual was alive then), or the last time they were Remembered (if not).\",\n"
+"            \"index\": 2,\n"
+"            \"type\": \"integer\"\n"
+"        },\n"
+"        \"flags\": {\n"
+"            \"binaryFormat\": \"I\",\n"
+"            \"description\": \"Other information about the individual: see 'flags'.\",\n"
+"            \"index\": 5,\n"
+"            \"type\": \"integer\"\n"
+"        },\n"
+"        \"pedigree_id\": {\n"
+"            \"binaryFormat\": \"q\",\n"
+"            \"description\": \"The 'pedigree ID' of this individual in SLiM.\",\n"
+"            \"index\": 1,\n"
+"            \"type\": \"integer\"\n"
+"        },\n"
+"        \"population\": {\n"
+"            \"binaryFormat\": \"i\",\n"
+"            \"description\": \"The population the individual was part of, either when the tree sequence was written out (if the individual was alive then), or the last time they were Remembered (if not).\",\n"
+"            \"index\": 3,\n"
+"            \"type\": \"integer\"\n"
+"        },\n"
+"        \"sex\": {\n"
+"            \"binaryFormat\": \"i\",\n"
+"            \"description\": \"The sex of the individual (hermaphrodite/male/female).\",\n"
+"            \"index\": 4,\n"
+"            \"type\": \"integer\"\n"
+"        }\n"
+"    },\n"
+"    \"required\": [\n"
+"        \"pedigree_id\",\n"
+"        \"age\",\n"
+"        \"population\",\n"
+"        \"sex\",\n"
+"        \"flags\"\n"
+"    ],\n"
+"    \"type\": \"object\"\n"
+"}";
+
+const std::string SLIM_TSK_POPULATION_METADATA_SCHEMA = 
```

Comment:
This *will* be confusing, the question is at which point does the terminology break happen. I think *this* should be called `SLIM_TSK_POPULATION_METADATA_SCHEMA` because it applies to the population table, and parallels eg `SLIM_TSK_NODE_METADATA_SCHEMA`. I'll change the words below to refer to 'subpopulations', though.

Reply:
OK, that seems reasonable